docs/source/programming_resources/tutorial_specific/android_studio/fork_and_clone_github_repository/Fork-and-Clone-From-GitHub.rst

```diff
@@ -12,26 +12,26 @@ Fork and Clone from GitHub
 Forks vs. Clones 
 ----------------
 
-A **Fork** on GitHub is a copy of another repository on GitHub from one account
+A `Fork <https://docs.github.com/en/get-started/quickstart/github-glossary#fork>`_ on GitHub is a copy of another `repository <https://docs.github.com/en/get-started/quickstart/github-glossary#repository>`_ on GitHub from one account
 to another account. The new forked repository retains a parent-child
-relationship with the origin repository. Forks are typically used when software
+relationship with the `origin <https://docs.github.com/en/get-started/quickstart/github-glossary#origin>`_ repository. Forks are typically used when software
 will have an independent line of development, such as when FTC teams develop
 their own team code using the `FIRST-Tech-Challenge/FtcRobotController <https://github.com/FIRST-Tech-Challenge/FtcRobotController>`_
 repository as a basis.  FTC teams should create a Fork of the
 `FIRST-Tech-Challenge/FtcRobotController <https://github.com/FIRST-Tech-Challenge/FtcRobotController>`_ repository as a convenient way to
 manage their software development process. Thanks to the parent-child
 relationship, when changes are made to the parent repository those changes can
-be easily tracked and fetched/merged into the forked repository, keeping the
+be easily tracked and `fetched <https://docs.github.com/en/get-started/quickstart/github-glossary#fetch>`_/`merged <https://docs.github.com/en/get-started/quickstart/github-glossary#merge>`_ into the forked repository, keeping the
 forked repository up to date.
 
 .. warning:: 
-   Teams should not issue pull requests against the upstream parent, the
+   Teams should not issue pull requests against the `upstream <https://docs.github.com/en/get-started/quickstart/github-glossary#upstream>`_ parent, the
    FIRST-Tech-Challenge/FtcRobotContoller repository. Forks of the
    FIRST-Tech-Challenge/FtcRobotContoller repo may always fetch changes, but
    should never attempt to push changes up to the repo.
 
-A **Clone** is a copy of a repository, typically on a local computer. A team
-member creates a branch of the team's repository for feature development, and
+A `Clone <https://docs.github.com/en/get-started/quickstart/github-glossary#clone>`_ is a copy of a repository, typically on a local computer. A team
+member creates a `feature branch <https://docs.github.com/en/get-started/quickstart/github-glossary#clone>`_ of the team's repository for feature development, and
```

Comment:
Did you mean for this to point to a glossary issue on a branch?

Reply:
Copy paste, ugh